dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -10,6 +10,7 @@ RUN apt-get update
 
 RUN apt-get install -f -y ifupdown arping libdbus-1-3 libdaemon0 libjansson4
 
+RUN apt-get install -f -y ndisc6
```

Comment:
can you add this into vs docker? we need to replicate same env in the vs.

Reply:
done.